cmd/minishift/cmd/start_preflight.go

```diff
@@ -62,10 +63,8 @@ func preflightChecksBeforeStartingHost() {
 			fmt.Sprintf("%s is not a valid OpenShift version", viper.GetString(configCmd.OpenshiftVersion.Name)),
 		)
 	} else {
+		fmt.Printf("FAIL\n")
 		fmt.Printf("-- Checking if requested OpenShift version '%s' is valid ... SKIP\n", viper.GetString(configCmd.OpenshiftVersion.Name))
-		if glog.V(2) {
-			fmt.Println("github.com is not accessible over internet or host is not connected to internet")
```

Comment:
This we put just in case if internet connectivity is there but `github.com` server is down. I don't this it's good to remove it just in case we want get --v 2 logs from user.

Reply:
The indication is still here. Have a look a few lines around this line. It will now instead append `FAIL` or `OK` to the end of the Internet connectivity check line. This is more consistent with how the other status lines work.